src/test/javascript/spec/service/orion/orion-assessment.service.spec.ts

```diff
@@ -4,7 +4,7 @@ import * as sinon from 'sinon';
 import { SinonStub, spy, stub } from 'sinon';
 import sinonChai from 'sinon-chai';
```

Comment:
This also uses sinon. I think it would make sense to fix this in this PR as well, wouldn't it?

Reply:
Thanks, totally missed that test